### DISCUSSION TITLE
Simple methods to override Model creation and update

### DISCUSSION THREAD

**Discussion by reinert** [encode/django-rest-framework#8028] [Ideas & Suggestions]
## Checklist

The ModelSerializer performs a lot of logic in create and update methods before and after actually saving the instance.

Thus, It would be good to have the option of overriding only the specific Model saving call.

I created a simple patch to explain better the idea, detaching two simple methods from create and update to allow users to customize only the actual saving logic.

https://github.com/encode/django-rest-framework/pull/7984

Suggestions are welcome.

**Comment by pradeepcep** [comment]
@reinert Can you please point out some use-cases where this might be helpful? For specifically overriding the model save call, I've seen projects usually overriding the model's own `save()` method. Do you see any cases where that might not be enough?
